libparsec/crates/crypto/src/sodiumoxide/sequester.rs

```diff
@@ -352,6 +352,7 @@ impl Serialize for SequesterVerifyKeyDer {
 
 impl SequesterVerifyKeyDer {
     const ALGORITHM: &str = "RSASSA-PSS-SHA256";
+    const DEFAULT_SIZE: usize = 4096 / 8;
```

Comment:
`DEFAULT_SIZE` correspond to the size of the verify key or the signature ?

Reply:
The RSA's signature

Reply:
Done